Knittery/Network/QueryDefines.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  QueryDefines.swift
+//  Knittery
+//
+//  Created by Nick on 2022-10-13.
+//  Copyright © 2022 Nicholas Valenti. All rights reserved.
+//
+
+import Foundation
+
+// notebook-p
+enum QNotebook: String {
+    case favorites = "faved"
+    case projects = "projects"
```

Comment:
You might already know this, but if the rawValue of a string backed enum is the same as the case, you don't actually have to define the rawValue. So in this case it would be fine to leave it as `case projects` and it infers the string value.
But I understand you might want to add it just for symmetry :D

Reply:
I didn't know this, in this case I'll leave it for clarity since Ravelry naming conventions can be a bit confusing